tests/onboarding/test_onboarding_install.py

```diff
@@ -2,9 +2,8 @@
 
```

Comment:
Could you remove this file name from `allow_no_feature_nodes` in pyptoject.toml ?

Reply:
done!